frontend/src/metabase/xray/xray.js

```diff
@@ -216,8 +97,16 @@ export default handleActions({
                 .assoc('comparison', payload)
                 .assoc('loading', false)
                 .value()
+    },
+    [XRAY_ERROR]: {
+        next: (state, { payload }) =>
+            chain(state)
+                .assoc('loading', false)
```

Comment:
I would prefer using the ES6 spread operator in these situations instead of underscore